### PR TITLE
pcreglow.github.io: journal-week-3

### PR DESCRIPTION
**Pull request by pcreglow**

### CODE REVIEW

**Comment by gatorpazz** [comment]
Good incorporation of what you got from the conference. Communication is definitely key. I like that you're demonstrating some git functions in the tutorial, perhaps do something crazy like explain merge conflicts with git? Good job! :+1:

**Comment by JessyRiordan** [comment]
Great job! 

Reflective: I like Wren's presentation as well. :+1: 

Tutorial: I am looking forward to reading the full tutorial. :+1:

**Comment by Epicurean306** [comment]
Like the topic choices of teamwork and command-line GitHub workflows. Solid drafts! :+1:

File: about/html.md [hunk 1]
```diff
@@ -0,0 +1,60 @@
+#[`HTML`](https://developer.mozilla.org/en-US/docs/Web/HTML)
+
+**`<tags>`**
```
Comment (al-the-x) [inline about/html.md]: Maybe use headings here instead?

File: about/html.md [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+[`<colgroup>`](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/colgroup) defines a group of columns within a `<table>`.
+
+[`<thead>`](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/thead) defines a set of rows, which defines the head of the columns of the table.
+
+[`<tfoot>`](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/tfoot) defines a set of rows, which summarizes the columns of the table. 
+
+[`<tbody>`](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/tbody) contains zero or more <tr> elements and is the body of its parent `<table>` elelemt.
+
+[`<tr>`](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/tr) defines a row of cells in a `<table>`. A `<tr>` elements can contain child elemens [`<td>`](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/td) and [`<th>`](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/th). The `<td>` element defines a cell of a table that contains data, while the `<th>` element defines a cell that is a header for a group of cells in a `<table>`.
+
```
Comment (al-the-x) [inline about/html.md]: More details and examples might be helpful to you... Great overall voice, though. Love the tutorial approach you're taking here...!

File: reflective-journal-week-3.md [hunk 3]
```diff
@@ -0,0 +1,11 @@
...
+
+Team work was a central themse throughout the conference and the presentations really struck a cord with me during the Friday sessions. It's easy to get lost in your own little world, and as an introvert, I often can slip into the habit of staying in a silo to accomplish a task or work assignment without soliciting feedback or assistance from others. Over the years, this has been a bad habit of mine and I was relieved to hear so many speakers reveal ways to overcome it. In particular, Wren Lanier's presentation - [How Designers Fail](http://www.slideshare.net/ModevHQ/wlanier-slides) - focused on common failures in communication and teamwork for designers and developers. The risk of failure can be avoided by:
+
+- A focus on learning and make it a part of every day
+- Be transparent by sharing your work in progress early and often
+- Put conversation with team members and clients at the center of your process
```
Comment (al-the-x) [inline reflective-journal-week-3.md]: Great takeaways, Phil. Don't forget these points as you move forward. The temptation will be to nose-down and Get Stuff Done, but you'll actually become _more_ efficient and effective by roping in your peers to help.

**Comment by al-the-x** [comment]
Need some more work on your Tutorial, but your reflective journals are always inspring. Great job.

**Pull request merged**